packages/suite/src/components/firmware/ReconnectInNormal.tsx

```diff
@@ -59,8 +59,8 @@ const Body = () => {
                     <Translation id="TR_DISCONNECT_YOUR_DEVICE" />
                 </H2>
                 <P>
-                    Device is in unexpected model. In the next step you will be asked to reconnect
-                    your device normally.
+                    <Translation id="TR_DEVICE_IS_IN_UNEXPECTED_MODE" />
+                    <Translation id="FIRMWARE_RECONNECT_NORMAL_NEXT_STEP_DESC" />
```

Comment:
Not sure why there are two translations in the same paragraph? Why not one? And there's also a dot missing in the first sentence.

Reply:
That unexpected mode strig appears two times in the project. So initially I wanted to reuse it, but later I realized that the other occurence was native. I'd say it is no harm to have these two splitted. 
Added the dot.

Reply:
Hmm, in this change it looks like `TR_DEVICE_IS_IN_UNEXPECTED_MODE` has been added. In any case, I think it makes more sense to me to have the whole paragraph in a single translation.

Reply:
:dagger: changes done